manifest/package.xml

```diff
@@ -61,5 +61,9 @@
         <!--<members>Student__c.CalculateGPA</members>-->
         <name>QuickAction</name>
     </types>
+    <types>
```

Comment:
Although using the * may not be an issue now, it can cause headaches in the future. For instance, when we enable communities, a bunch of Lightning Components and their respective classes will be created. If the * is used when retrieving from the org, you'll also grab all of those files which is not ideal.

Reply:
Thank you for the helpful feedback. I will clean the code up before my next commit.